### PR TITLE
Translate block labels

### PR DESCRIPTION
**Pull request by core23**
* [x] German
* [x] English
* [x] French

### CODE REVIEW

**Comment by core23** [comment]
Please help me with the translation @rande @Soullivaneuh @sagikazarmark

File: Resources/translations/SonataPageBundle.de.xliff [hunk 1]
```diff
@@ -798,6 +798,26 @@
...
+          <source>form.label_current</source>
+          <target>Aktuelle Seite</target>
+      </trans-unit>
+      <trans-unit id="202" resname="form.label_page">
+          <source>form.label_page</source>
+          <target>Seite</target>
+      </trans-unit>
+      <trans-unit id="203" resname="form.label_class">
+          <source>form.label_class</source>
+          <target>Class</target>
```
Comment (OskarStark) [inline Resources/translations/SonataPageBundle.de.xliff]: Klasse?

**Comment by OskarStark** [comment]
Do you have a screenshot where these translations are used?

**Comment by core23** [comment]
They are used when you create a SharedBlockBlockService or ChildrenPagesBlockService via the page composer.

File: Resources/translations/SonataPageBundle.de.xliff (hunk 1 above)
Comment (OskarStark) [inline Resources/translations/SonataPageBundle.de.xliff]: ``CSS Klasse``?

**Comment by OskarStark** [comment]
Thank you @core23

**Comment by core23** [comment]
@rande @Soullivaneuh Can you help with the French translation so we can merge this.

**Comment by rande** [comment]
form.label_block => Bloc 
 form.label_title => Titre 
 form.label_current => Page actuelle
 form.label_page => Page
 form.label_class => Class CSS
 form.label_text => Text
 form.label_cache_policy => Méthode de cache
 form.label_cache_policy_public =>  Public
 form.label_cache_policy_private => Privé
 form.label_menu_name => Nom du menu
 form.label_safe_labels => Labels
 form.label_current_class => Class CSS élément séléctionné
 form.label_first_class => Class CSS premier élément
 form.label_last_class => Class CSS dernier élément 
 form.label_menu_class => Class CSS Menu
 form.label_children_class => Class CSS enfant
 form.label_menu_template => Modèle de page

**Comment by core23** [comment]
Rebased

**Comment by core23** [comment]
RTM @OskarStark

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**